server/sonar-web/src/main/js/components/issue/collections/issues.js

```diff
@@ -26,8 +26,7 @@ define([
       return r.issues.map(function (issue) {
         var component = find(r.components, issue.component),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ASemicolon)